### ISSUE TITLE
Filtering rows by content found in array of columns that are not visible doesn't works

### ISSUE THREAD

**Issue by ZaDarkSide** [mediakular/gridcraft#3]
Trying to filter rows by an array of columns that are not visible will not work, there are some cases where you want to filter the rows by content found in an invisible column. Right now how it is implemented it will filter row if it finds a match in a visible column only.
The `all` case should filter for visible columns like it is now, but if I specify an array of columns even if they are not visible it should allow me to filter by them.

https://github.com/mediakular/gridcraft/blob/28870ddb7dbe745eb74ed42623e8378a0aa9ab15/src/lib/GridFunctions.ts#L51-L59

**Comment by st3v3y** [comment]
Agreed. I already adapted the behavoir, so invisible columns can be filtered, too. Thanks for the heads-up!
